recipes/extract-sv-reads/meta.yaml

```diff
@@ -4,22 +4,26 @@ build:
 
 package:
   name: extract-sv-reads
-  version: '1.1.2'
+  version: '1.2.1'
 source:
   # Uses git_url to produce correct versions in binary
   # https://github.com/bioconda/bioconda-recipes/issues/5990
   git_url: https://github.com/hall-lab/extract_sv_reads.git
-  git_tag: 9bb74fc82d44f1abd73033cbb2e11ba6cf59f885
+  git_tag: cc0507aceb9c48df083c5ed034f5f246ea5ee1d7
 
 requirements:
   build:
     - gcc
     - cmake
     - perl
-    - zlib
+    - curl
+    - bzip2 {{ CONDA_BZIP2 }}*
+    - xz
   run:
     - libgcc
-    - zlib
+    - curl
+    - bzip2 {{ CONDA_BZIP2 }}*
+    - xz
```

Comment:
can we pin xz 5.2.* as well

Reply:
Sure thing. What's required to do that? Add a line to env_matrix.yml like so?
```
CONDA_XZ: "5.2"
```

Reply:
Yes, this would be supercool!